run.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
```

Comment:
Could we have an additional script for remote-dev? or an if else block if we put MULTISTAGE_TARGET="remote-development"?

Reply:
Yes, readme and scripts should be updated once we agree on the new usage, but I didn't want to change too much in one go, hence why these scripts are still present